packages/category-feed-luigis-box/phpunit.xml

```diff
@@ -2,16 +2,14 @@
 <!-- http://www.phpunit.de/manual/current/en/appendixes.configuration.html -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     backupGlobals="false"
-    backupStaticAttributes="false"
     colors="true"
-    convertErrorsToExceptions="true"
-    convertNoticesToExceptions="true"
-    convertWarningsToExceptions="true"
     processIsolation="false"
     stopOnFailure="false"
     failOnWarning="true"
     beStrictAboutTestsThatDoNotTestAnything="false"
-    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.1/phpunit.xsd"
+    cacheDirectory=".phpunit.cache"
```

Comment:
I noticed we install phpunit 11.2 but reference 11.1 here. I do not know it is an issue and if we want to fix that :shrug: It is the same in the latest branch, just saying